doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -63,6 +63,7 @@ the following specifications:
 |`ra2.ch.015`|Physical NIC Speed - Network Intensive Profile|The NIC ports housed in the physical machines on which the Kubernetes Nodes run for workloads matching the Network Intensive profile must be at least 25Gbps.|[infra.hw.nic.cfg.002](./chapter02.md#224-cloud-infrastructure-hardware-profile-requirements)|
 |`ra2.ch.016`|Physical PCIe slots|The physical machines on which the Kubernetes Nodes run must be equipped with at least eight (8) Gen3.0 PCIe slots, each with at least eight (8) lanes.|
 |`ra2.ch.017`|Immutable infrastructure|Whether physical or virtual machines are used, the Kubernetes Node is not changed after it is made ready for use. New changes to the Kubernetes Node are rolled out as new instances. This covers any changes from BIOS through Operating System to running processes and all associated configurations.|[req.gen.cnt.02](./chapter02.md#23-kubernetes-architecture-requirements)|[Use of Ansible](../../../ref_impl/cntt-ri2/chapters/chapter04.md#431-installation-on-bare-metal-infratructure)|
+|`ra2.ch.018`|NFD|[Node Feature Discovery](https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html) must be used to advertise the detailed software and hardware capabilities of each node in the Kubernetes Cluster.|TBD|TBD|
```

Comment:
Agreed that NFD should be used. DO you want to state that HW accelerators should/must have NFD container so its capabilities are discovered and recorded as node features?

Reply:
Yes, this is the intention without going to the details of NFD containers.

Reply:
@arkadykanevsky do I need to change something?